cmd/ext.go

```diff
@@ -0,0 +1,59 @@
+package cmd
+
+import (
+	"github.com/lunarway/shuttle/internal/extensions"
+	"github.com/spf13/cobra"
+)
+
+func newExtCmd() *cobra.Command {
+	extManager := extensions.NewExtensionsManager("some registry")
```

Comment:
It seems wrong to hard code a string here?

Reply:
Will be replace in a further pr. Won't fix